12LessonTwelve/homework/index.html

```diff
@@ -20,7 +20,7 @@ <h1 class="section-title">Best Selling Products</h1>
         </div>
     </header>
     <section class="products">
-        <div class="col-lg-3 col-sm-6 col-xs-12 text-center patient">
+        <div class="col-lg-10 col-sm-8 col-xs-12 text-center patient">
             <article>
                 <figure>
                     <a href="#">
```

Comment:
Something funky is going here with that first div. Please fix and I can bump your grade up.
<img width="1154" alt="screen shot 2016-09-13 at 5 54 46 pm" src="https://cloud.githubusercontent.com/assets/12698279/18494352/3a89f2c0-79db-11e6-883f-c899103b9e19.png">